src/app/components/GearCardConnection/GearCardConnection.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Cable_1 from '../assets/cables/Cable1_Icon';
+import Cable_2 from '../assets/cables/Cable2_Icon';
+import Cable_3 from '../assets/cables/Cable3_Icon';
+import Cable_4 from '../assets/cables/Cable4_Icon';
+import Cable_5 from '../assets/cables/Cable5_Icon';
+import Cable_6 from '../assets/cables/Cable6_Icon';
+import Cable_7 from '../assets/cables/Cable7_Icon';
+import Typography from '../Typography/Typography';
+import style from './GearCardConnection.module.css';
+
+type GearConnectionProps = {
+  connection: string;
+  channel: string;
+};
+
+function getRandomCable(min: number, max: number) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1) + min);
+}
+
+const cables = [
+  <Cable_1 fill="var(--navigation-icon-color)" />,
+  <Cable_2 fill="var(--navigation-icon-color)" />,
+  <Cable_3 fill="var(--navigation-icon-color)" />,
+  <Cable_4 fill="var(--navigation-icon-color)" />,
+  <Cable_5 fill="var(--navigation-icon-color)" />,
+  <Cable_6 fill="var(--navigation-icon-color)" />,
+  <Cable_7 fill="var(--navigation-icon-color)" />,
+];
+
+export default function Cable1({
```

Comment:
This also seems to be incorrectly named :)

Reply:
Resolved in [76b9e9b](https://github.com/Seborider/plugs./pull/33/commits/76b9e9b7203ba3524422a0f2dfbae26dfe148aa8)